### PR TITLE
jQuery.SimpleSelect

### PR DESCRIPTION
**Pull request by Exacore**

### CODE REVIEW

File: files/jquery.simpleselect/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "jquery.simpleselect",
+  "repo": "pioul/jQuery.SimpleSelect",
+  "files": {
+    "include": ["jquery.simpleselect.css","jquery.simpleselect.js","jquery.simpleselect.min.css","jquery.simpleselect.min.js"]
```
Comment (jimaek) [inline files/jquery.simpleselect/update.json]: just the min files are enough for production

Comment (Exacore) [inline files/jquery.simpleselect/update.json]: Hi,

Is it okay now ?

Thanks

File: files/jquery.simpleselect/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "jquery.simpleselect",
+  "repo": "pioul/jQuery.SimpleSelect",
+  "files": {
+    "include": ["jquery.simpleselect.min.js"]
```
Comment (jimaek) [inline files/jquery.simpleselect/update.json]: you can use `*.min.*` to include all the min automatically. You are missing the css right now

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**